src/main/java/frc/robot/subsystems/Tabs/TabContainer.java

```diff
@@ -8,11 +8,12 @@
 
 /** Add your docs here. */
 public class TabContainer {
-
   private TabDriveBase mTabDriveBase;
 
   public TabContainer(DriveBaseSubsystem d) {
-    mTabDriveBase = new TabDriveBase(d);
+    if (d != null) {
+      mTabDriveBase = new TabDriveBase(d);
+    }
```

Comment:
I think the drive base team needs to come up with a more comprehensive solution here. Unfortunately this won't prevent the null reference issues in Robot.java